plugins/datastreamer_plugin/ui/topic_selection_dialog/dialogselecttopics.cpp

```diff
@@ -305,13 +311,27 @@ void DialogSelectTopics::reset_to_configuration_()
 
     // Add XML files
     // First erase every data in the list
-    ui->xml_files_list->clear();
     for (const auto& file : configuration_.xml_datatypes_files)
     {
         // Add every xml file in configuration
         ui->xml_files_list->addItem(file);
     }
 
+    // Add every xml file stored already in configuration
+    ui->xml_files_list->clear();
+    for (const auto& xml_file : configuration_.xml_datatypes_files)
+    {
+        try
+        {
+            add_xml_file_(xml_file);
+        }
+        catch (const Exception& e)
```

Comment:
NIT: Maybe we can be more specific here about the exception type (taking advantage of the several exception types we have), to make sure we are not absorbing exceptions that should be propagated upstream